README.md

```diff
@@ -73,6 +74,7 @@ A curated list of awesome companies that offer their tools and services for free
 
 - [BugSnag](https://www.bugsnag.com/) `requires-approval` - Exception tracking and reporting.
 - [Honeybadger](https://www.honeybadger.io) `requires-approval` `non-commercial` - Exception and uptime monitoring for Ruby.
+- [Sentry](https://sentry.io) - Exception tracking and reporting
```

Comment:
Sentry as a product is open source, but as far as I can tell they don't actually offer any specific benefits to other open source projects. This usually comes in the form of specific, open source pricing (which is separate from a free tier that anyone can use). Does Sentry offer any price breaks or benefits to other open source projects?

Reply:
No. If that's the criteria, I misunderstood. I assumed that offering it as OSS also counts as "OSS supporter"

Reply:
@zachflower Do you want me to update the PR and remove Sentry?